DuckDuckGo/MainViewController.swift

```diff
@@ -137,6 +137,8 @@ class MainViewController: UIViewController {
     
     // Skip SERP flow (focusing on autocomplete logic) and prepare for new navigation when selecting search bar
     private var skipSERPFlow = true
+        
+    private var keyboardHeight: CGFloat = 0.0
```

Comment:
Set when the keyboard frame changes so that it can be used when refreshing the content insets for the web view